datasets/gov/vstt/saugomu_ter_kadastras.csv

```diff
@@ -0,0 +1,718 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vstt/teritoriju_kadastras,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,BuveiniuApsaugaiSvarbiTeritorija,,,id,,,4,,,Buveinių apsaugai svarbios teritorijos,
+,,,,,id,string,,,,4,open,,Identifikavimo kodas,
+,,,,,site_code,string,,,,2,open,,Vietovės identifikatorius,
+,,,,,gkodas,string,,,,2,open,,InGIS geokodas (saugomos teritorijos tipas),
+,,,,,vieta,string,,,,2,open,,Vieta,"Vieta aprašomoji informacija (gali būti nurodoma savivaldybė, seniūnija, jei miškas – papildomai miškų urėdija, girininkija)"
+,,,,,registruota,date,D,,,4,open,,"Įregistravimo į kadastrą data (metai, mėnuo, diena)",
+,,,,,id_vart,integer,,,,2,open,,Identifikatorius, Institucija/-jos atsakinga/-os už priežiūrą ir monitoringą. Duomenų teikėjų klasifikatorius
+,,,,,pastabos,string,,,,2,open,,Pastabos,
+,,,,,pavadinimas,string,,,,4,open,,Buveinės,Buveinių  apsaugai svarbios  teritorijos pavadinimas
+,,,,,plotas,number,ha,,,4,open,,Plotas,Juridinis
+,,,,,ilgis,number,km,,,4,open,,Ilgis,Kai nenurodomas plotas
+,,,,,statusas_bast,integer,,,,2,open,,Apsaugos statusas,
+,,,,,statuso_ta,string,,,,4,open,,Aktas,Buveinių apsaugai svarbios vietovės įsteigimo pagrindas (juridinis aktas)
+,,,,,bendr_reglam,string,,,,4,open,,Reglamentas,"Bendrieji veiklos reglamentai, nuoroda į teisės aktą"
+,,,,,steigiama,date,D,,,4,open,,Statuso suteikimo data,
+,,,,,steig_tikslas,string,,,,2,open,,Tikslas,Saugomos teritorijos priskyrimo Natura 2000 tinklui tikslas (aprašomoji informacija)
+,,,,,geometrija,geometry,,,,4,open,,Ploto geometrija,
+,,,,,,comment,type,,"update(type: ""geometry(linestring, 3346)"")",3,open,,,
+,,,,,,,,,,,,,,
+,,,,BiosferosPoligonas,,,id,,,4,,,Biosferos poligonai,
+,,,,,id,string,,,,4,open,,Identifikavimo kodas,
+,,,,,pavadinimas,string,,,,4,open,,Biosferos poligono pavadinimas,
+,,,,,gkodas,string,,,,2,open,,InGISgeokodas (saugomos teritorijos tipas),
+,,,,,plotas,number,ha,,,4,open,,Plotas, 
+,,,,,steigejas,string,,,,2,open,,Steigėjas – institucija,
+,,,,,steigimas,date,D,,,4,open,,Steigimo data,
+,,,,,steig_ta,string,,,,4,open,,Įsteigimo teisės aktas,
+,,,,,steig_tikslas,string,,,,3,open,,"Steigimo tikslas,Aprašomoji informacija",
+,,,,,bendr_reglam,integer,,,,4,open,,Bendrieji veiklos reglamentai,
+,,,,,spec_reglam,string,,,,4,open,,Specialieji / individualieji veiklos reglamentai,
+,,,,,saltiniai,string,,,,2,open,,Duomenų  šaltiniai (nurodomi moksliniai tyrimai ir publikacijos),
+,,,,,tarpt_svarba_status,integer,,,,4,open,,Teritorijos statusas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statusas
```

Comment:
```suggestion
,,,,,tarpt_svarba_status,integer,,,,2,open,,Teritorijos statusas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statusas
```
Duomenys pateikti kodais, nėra atskirai pateiktos duomenų reikšmės ar klasifikatorius.
https://atviriduomenys.readthedocs.io/dsa/dimensijos.html#enum

Jau buvo pateiktas šis siūlymas, tačiau jis nebuvo priimta, kodėl?

Reply:
Pagal pateiktus `description` atrodytų, kad `tarp_svarba_status` aprašymas nėra pateiktas `tarpt_svarba_apras`.
Jeigu taip, tuomet duomenys yra denormalizuoti ir brandos lygis turėtų būti 2.